Vizzuality/heco-invest
# feat: Enhancing project export at Backoffice

This PR enhances `ProjectExporter` which is responsible for generating `csv` file returned to user during export via Backoffice so it returns all required fields defined at task - basically all fields that are visible at Backoffice form needs to be also part of exported csv.

## Testing instructions

Login as Admin and go to the Projects section. Click on Export csv and verify that:
 - csv is exported
 - it contains all projects
 - for every project, all attributes defined at task are exported

## Tracking

https://vizzuality.atlassian.net/browse/LET-1379
